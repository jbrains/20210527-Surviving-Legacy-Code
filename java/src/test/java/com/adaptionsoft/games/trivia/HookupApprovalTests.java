package com.adaptionsoft.games.trivia;

import org.approvaltests.Approvals;
import org.junit.Test;

import java.util.Arrays;

public class HookupApprovalTests {
    @Test
    public void isThisThingOn() throws Exception {
        Approvals.verifyAll("testing", Arrays.asList(1, 2, 3, 4, 5));
    }
}
