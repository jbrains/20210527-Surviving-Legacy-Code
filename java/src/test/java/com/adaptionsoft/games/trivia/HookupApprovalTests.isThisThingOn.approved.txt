testing[0] = 1
testing[1] = 2
testing[2] = 3
testing[3] = 4
testing[4] = 5
